Catappult/appcoins-wallet-android
# [APPC 3082] Simplify home crypto

**What does this PR do?**
Home UI changes:
 - Removed send/receive button from Home 
 - New UI separator on balance 
 - Only showing Appc-credits on Home 

**Where should the reviewer start?**
Mostly home layouts.

**How should this be manually tested?**
 - Check Home UI in portrait/landscape mode 
 - Check Home with a notification

**What are the relevant tickets?**
https://aptoide.atlassian.net/browse/APPC-3083

**Code Review Checklist**
